reports/transmissibility.Rmd

```diff
@@ -559,6 +559,47 @@ last_trends %>%
 
 ## Results
 
+```{r}
+library(R0)
+
+mGT <- generation.time("gamma", c(3, 1.5))
+
+# Helper function to get R0 outputs in a tidy format, compatible with pipelines
+r0_quantiles <- function(Rt) {
+  
+  data.frame(
+    q50   = Rt$R,
+    q2.5  = Rt$conf.int$lower,
+    q97.5 = Rt$conf.int$upper
+  )
+  
```

Comment:
Just to put a pin in this: also will need to standardize notations for quantiles, confidence intervals, credibility intervals, medians